cmd/config.go

```diff
@@ -143,24 +143,33 @@ var configAddCmd = &cobra.Command{
 			}
 			err := viper.WriteConfigAs(filepath.Join(configFilePath, configName+configExtension))
 			if err != nil {
-				output.RenderError(err.Error(), outputOptions)
-				os.Exit(1)
+				return err
 			}
-			resultData := output.Result{
-				Result: "success",
-				ResultData: map[string]interface{}{
-					"lagoon":   lagoonConfig.Lagoon,
-					"hostname": lagoonConfig.Hostname,
-					"graphql":  lagoonConfig.GraphQL,
-					"port":     lagoonConfig.Port,
-					"ui":       lagoonConfig.UI,
-					"kibana":   lagoonConfig.Kibana,
+			output.RenderOutput(output.Table{
+				Header: []string{
+					"Name",
+					"GraphQL",
+					"SSH-Hostname",
+					"SSH-Port",
+					"UI-URL",
+					"Kibana-URL",
 				},
-			}
-			output.RenderResult(resultData, outputOptions)
+				Data: []output.Data{
+					[]string{
+
+						lagoonConfig.Lagoon,
+						lagoonConfig.GraphQL,
+						lagoonConfig.Hostname,
+						lagoonConfig.Port,
+						lagoonConfig.UI,
+						lagoonConfig.Kibana,
+					},
+				},
+			}, outputOptions)
 		} else {
 			output.RenderError("Must have Hostname, Port, and GraphQL endpoint", outputOptions)
```

Comment:
This will continue to the `return nil` below.. I think you might be able to just do something like this instead to get the standard cobra error formatting:
```
return fmt.Errorf("Must have Hostname, Port, and GraphQL endpoint")
```